BUILDING.md

```diff
@@ -60,12 +60,23 @@ Node.js relies on V8 and libuv. We adopt a subset of their supported platforms.
 There are three support tiers:
 
 * **Tier 1**: Full test coverage and maintenance by the Node.js core team and
-  the broader community.
+  the broader community. All commits to the Node.js repository are tested on
+  multiple variants of these platforms. Breakages are considered critical for
+  the purpose of creating releases.
 * **Tier 2**: Full test coverage. Limited maintenance, often provided by the
-  vendor of the platform.
-* **Experimental**: May not compile or test suite may not pass.
-  These are often approaching Tier 2 support but are not quite ready.
-  There is at least one individual providing maintenance.
+  vendor of the platform. All commits to the Node.js repository are generally
+  tested on multiple variants of these platforms but it is not considered
+  essential if there are temporary problems with the test infrastructure.
+  Breakages are generally considered critical for the purpose of creating
```

Comment:
Drop "generally", "considered", and "the purpose of" here (and probably everywhere else), similar to above comment. I'm also not sure what "critical" means here. Maybe wording as the positive will help? Instead of "Breakages are critical for creating releases", it could be "Passing tests are required when creating releases."?